src/display/annotation_layer.js

```diff
@@ -480,22 +481,28 @@ class TextWidgetAnnotationElement extends WidgetAnnotationElement {
         element.classList.add("comb");
         element.style.letterSpacing = `calc(${combWidth}px - 1ch)`;
       }
+      if (
```

Comment:
I don't think we actually want this in edit mode. Edit mode has other styles to clearly differentiate it from display mode. In display mode we do want this, and that's why it only lived in the `else` clause. The objective of this patch is also to make the final rendering on display/print/save in the right style.